src/test/app/AccountDelete_test.cpp

```diff
@@ -912,6 +912,275 @@ class AccountDelete_test : public beast::unit_test::suite
         env.close();
     }
 
+    void
+    testDestinationDepositAuthCredentials()
+    {
+        {
+            testcase(
+                "Destination Constraints with DepositAuth and Credentials");
+
+            using namespace test::jtx;
+
+            Account const alice{"alice"};
+            Account const becky{"becky"};
+            Account const carol{"carol"};
+
+            const char credType[] = "abcd";
+
+            Env env{*this};
+            env.fund(XRP(100000), alice, becky, carol);
+            env.close();
+
+            // carol issue credentials for becky
+            env(credentials::create(becky, carol, credType));
+            env.close();
+
+            // get credentials index
+            auto const jv =
+                credentials::ledgerEntryCredential(env, becky, carol, credType);
+            std::string const credIdx = jv[jss::result][jss::index].asString();
+
+            // Close enough ledgers to be able to delete becky's account.
+            incLgrSeqForAccDel(env, becky);
+
+            auto const acctDelFee{drops(env.current()->fees().increment)};
+
+            // becky use credentials but they aren't accepted
+            env(acctdelete(becky, alice),
+                credentials::IDs({credIdx}),
+                fee(acctDelFee),
+                ter(tecBAD_CREDENTIALS));
+            env.close();
+
+            {
+                // alice sets the lsfDepositAuth flag on her account.  This
+                // should prevent becky from deleting her account while using
+                // alice as the destination.
+                env(fset(alice, asfDepositAuth), fee(drops(10)));
+                env.close();
+            }
+
+            // Fail, credentials doesn’t belong to carol
+            env(acctdelete(carol, alice),
```

Comment:
```suggestion
            // Fail, credentials doesn’t belong to alice
            env(acctdelete(alice, carol),
```
I'm assuming this test has a slight mistake since it isn't quite testing what the comment says it is.

Reply:
No, it is correct. Carol delete her account, but use credentials that belong to Becky